# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1277235596 25200
#      Tue Jun 22 12:39:56 2010 -0700
# Node ID 998d7ba68009ae9d4c9bbe03671bdac2a499b163
# Parent  e54152088b0ab04012d5d05695ef4321cba0ed56
Correctly break lines > 80 characters in ChangeLog.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-06-16  Rik <octave@nomad.inbox5.com>
+
+        * ChangeLog: Correctly break lines > 80 characters.
+
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
         * interpreter/intro.txi: Clarify wording.  (bug #30165).
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
         * interpreter/set.txi: Remove reference to deprecated function.
 
@@ -88,18 +92,18 @@ 2010-01-17  Thorsten Meyer  <thorsten.me
 
 2010-01-09  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/munge-texi.cc: Avoid C-style casts.
 	(main): Call usage if fewer than two arguments.
 
 2009-12-20  Rik <octave@nomad.inbox5.com>
 
-	* interpreter/geometryimages.m: Correctly set postscript output when QHULL
-	library does not exist and sombrero image is printed.
+	* interpreter/geometryimages.m: Correctly set postscript output when 
+        QHULL library does not exist and sombrero image is printed.
 
 2009-12-19  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am, faq/Makefile.am, interpreter/Makefile.am,
 	liboctave/Makefile.am, refcard/Makefile.am, interpreter/images.awk: 
 	Eliminate TOPDIR variable in favor of built-in automake variables of 
 	top_builddir and top_srcdir.
 
@@ -109,71 +113,72 @@ 2009-12-17  Rik  <octave@nomad.inbox5.co
 
 2009-12-11  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: remove install-local target and use automake 
 	syntax to install doc-cache file.
 
 2009-12-10  Rik <octave@nomad.inbox5.com>
 
-	* interpreter/Makefile.am: Replace distdir-local with dist-hook rule as it
-	is better supported by automake.
+	* interpreter/Makefile.am: Replace distdir-local with dist-hook rule as
+        it is better supported by automake.
 
 2009-12-10  Rik <octave@nomad.inbox5.com>
 
-	* interpreter/Makefile.am: new distdir-local rule to check for presence of
-	necessary libraries before packaging distribution
+	* interpreter/Makefile.am: new distdir-local rule to check for presence
+        of necessary libraries before packaging distribution
 
 2009-12-08  Rik <octave@nomad.inbox5.com>
 
-	* interpreter/Makefile.am: Distribute all generated documentation (.pdf,
-	.ps, .html, .dvi) to remove build requirement for TexInfo
+	* interpreter/Makefile.am: Distribute all generated documentation 
+        (.pdf, .ps, .html, .dvi) to remove build requirement for TexInfo
 
 2009-12-07  Rik <octave@nomad.inbox5.com>
 
-	* interpreter/Makefile.am, faq/Makefile.am, liboctave/Makefile.am: Document
-	use of NOTPARALLEL directive.
+	* interpreter/Makefile.am, faq/Makefile.am, liboctave/Makefile.am: 
+        Document use of NOTPARALLEL directive.
 
 2009-12-07  Rik <octave@nomad.inbox5.com>
 
-	* refcard/Makefile.am: Add CLEANFILES and MAINTAINERCLEANFILES objects to
-	properly clean directory.
+	* refcard/Makefile.am: Add CLEANFILES and MAINTAINERCLEANFILES objects
+        to properly clean directory.
 
 2009-12-05  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Set dependency of *.texi files on
 	munge-texi source code rather than executable.
 
 2009-12-04  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/images.awk: Revert to using Ghostscript for documentation
 	pdf image creation rather than Octave.
 
 2009-12-04  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am, interpreter/Makefile.am, liboctave/Makefile.am: remove
-	  lingering configuration support for conf.texi
-	* liboctave/Makefile.am: distribute .texi files
+	  lingering configuration support for conf.texi.
+	* liboctave/Makefile.am: distribute .texi files.
 
 2009-12-04  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/geometryimages.m, interpreter/interpimages.m,
 	  interpreter/sparseimages.m, interpreter/plotimages.m:
-	  Switch to color output for PostScript documentation images
+	  Switch to color output for PostScript documentation images.
 
 2009-12-04  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/geometryimages.m, interpreter/interpimages.m,
 	  interpreter/sparseimages.m, interpreter/plotimages.m:
-	  Set default paperorientation to landscape to stop warning message
+	  Set default paperorientation to landscape to stop warning message.
 
 2009-12-04  Ben Abbott <bpabbott@mac.com>
 
 	* interpreter/geometryimages.m, interpreter/interpimages.m,
-	  interpreter/sparseimages.m: Set default papersize to the desired imagesize
+	  interpreter/sparseimages.m: Set default papersize to the desired 
+          imagesize.
 
 2009-12-04  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/images.awk: Emit rules to copy PNG files to
 	octave.html directory once it is created.
 	* interpreter/Makefile.am (all-local): Depend on $(HTMLDIR_IMAGES).
 	(IMAGES): Include $(HTMLDIR_IMAGES) in the list.
 
@@ -240,18 +245,18 @@ 2009-12-03  John W. Eaton  <jwe@octave.o
 
 2009-12-02  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Correct Makefile.am to allow VPATH builds of
 	documentation
 
 2009-12-01  Rik <octave@nomad.inbox5.com>
 
-	* faq/Makefile.am, doc/interpreter/bugs.txi, doc/interpreter/install.txi:
-	Eliminate false dependency on conf.texi
+	* faq/Makefile.am, doc/interpreter/bugs.txi, 
+        doc/interpreter/install.txi: Eliminate false dependency on conf.texi.
 
 2009-11-30  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
 	Overhaul Makefile to correctly generate *.texi files and to stop their
 	distribution when running 'make dist'
 
 2009-11-30  John W. Eaton  <jwe@octave.org>
@@ -282,40 +287,41 @@ 2009-11-27  David Bateman  <dbateman@fre
 
 	* interpreter/install.txi: Document that curl libraries impact ftp
 	objects.
 	* interpreter/system.txi: Document the ftp objects
 
 2009-11-25  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
-	Simplify doc-cache handling in Makefile.am to use only install-data-local
-	target
+	Simplify doc-cache handling in Makefile.am to use only 
+        install-data-local target
 
 2009-11-24  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
-	Correct Makefile so that it halts on error when unable to build doc-cache
+	Correct Makefile so that it halts on error when unable to build 
+        doc-cache.
 
 2009-11-23  Rik <octave@nomad.inbox5.com>
 
 	* faq/Makefile.am, interpreter/Makefile.am, liboctave/Makefile.am:
 	Stop distribution of conf.texi by autotools.  conf.texi must be re-made
 	from conf.texi.in during configure step
 
 2009-11-23  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Add . to SUBDIRS to force automake to compile this
-	directory, and make conf.texi, before recursively compiling SUBDIRS which
-	depend on conf.texi
+	directory, and make conf.texi, before recursively compiling SUBDIRS
+        which depend on conf.texi
 
 2009-11-11  Rik <octave@nomad.inbox5.com>
 
-	* Makefile.am: Add DISTCLEANFILES macro so that generated conf.texi can be
-	cleaned by automake.
+	* Makefile.am: Add DISTCLEANFILES macro so that generated conf.texi can
+        be cleaned by automake.
 
 2009-11-10  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.am, faq/Makefile.am, interpreter/Makefile.am,
 	liboctave/Makefile.am, refcard/Makefile.am,
 	interpreter/config-images.sh, interpreter/images,
 	interpreter/images.awk: New files.
 	* Makefile.in, faq/Makefile.in, interpreter/Makefile.in,
@@ -412,20 +418,20 @@ 2009-06-07  Rik  <rdrider0-list@yahoo.co
 	Updated functions delete, allchild, backend, clf, hold, and refreshdata.
 
 2009-06-07  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* interpreter/container.txi: Udate documentation of container types
 
 2009-06-03  Rik  <rdrider0-list@yahoo.com>
 
-	* interpreter/plotimages.m: Correct errorbar plot for docs.  Make hist plot
-	reprodroducible for docs.
-	* interpreter/interpimages.m: Correct typo in legend of interpft plot for
-	docs
+	* interpreter/plotimages.m: Correct errorbar plot for docs.  Make hist
+        plot reprodroducible for docs.
+	* interpreter/interpimages.m: Correct typo in legend of interpft plot
+        for docs.
 
 2009-06-03  Jaroslav Hajek  <highegg@gmail.com>
 
 	* Makefile.in: Omit conf.texi in DISTFILES.
 
 2009-06-02  Jaroslav Hajek  <highegg@gmail.com>
 
 	* interpreter/oop.txi: Update docs of polynomial class, mention
@@ -445,28 +451,29 @@ 2009-05-25  Rik  <rdrider0-list@yahoo.co
 	(DIST_FILES) so that pdfs will build.
 
 2009-05-24  Robert T. Short  <octave@phaselockedsystems.com>
 
 	* interpreter/oop.txi: Added inheritance documentatin
 
 2009-05-21  Rik  <rdrider0-list@yahoo.com>
 
-	* interpreter/Makefile.in, texmf.cnf: Change texi2pdf call to include local
-	TeX configuration file which increases save_history variable.
+	* interpreter/Makefile.in, texmf.cnf: Change texi2pdf call to include
+        local TeX configuration file which increases save_history variable.
 
 2009-05-17  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/tips.txi: Change recommendation to use @tex rather than
 	@iftex construction.
 
 2009-05-17  Rik  <rdrider0-list@yahoo.com>
 
-	* interpreter/*.txi: Simplify TeXinfo files by eliminating redundant @iftex
-	followed by @tex construction.  Also, spellchecked all .txi and .texi files.
+	* interpreter/*.txi: Simplify TeXinfo files by eliminating redundant
+        @iftex followed by @tex construction.  Also, spellchecked all .txi 
+        and .texi files.
 
 2009-05-14  Rik  <rdrider0-list@yahoo.com>
 
 	* interpreter/arith.txi: Update section 17.8 (Mathematical Consants)
 	of arith.txi.
 
 2009-05-01  Rik  <rdrider0-list@yahoo.com>
 
